src/doc/reference.md

```diff
@@ -3316,6 +3316,9 @@ assert!(b != "world");
 assert!(p.0 == 10);
 ```
 
+For historical reasons and convenience, the tuple type with no elements (`()`)
+is often called "unit" or "the unit type."
```

Comment:
Can you 

1. use single instead of double curlies
2. put the period outside of the single curlie
3. use actual curlie quotes rather than `'` (there are some in this file if you want to copy/paste them.